README.md

```diff
@@ -39,12 +39,57 @@ I'm trying to keep it light so as not to slow down your render function. I want
 
 ## Installation
 
-You can install the package `react-faux-dom` from npm as you usually would. Then use webpack or browserify (etc) to bundle the source into your build. If you need a pre-built UMD version you can use [npmcdn][].
-
-You can find the latest version of the UMD version at https://npmcdn.com/react-faux-dom/dist/ReactFauxDOM.min.js
+You can install the package `react-faux-dom` from npm as you usually would. Then use webpack or browserify (etc) to bundle the source into your build. 
+If you need a pre-built UMD version you can use [npmcdn][]. You can find the latest version of the UMD version at https://npmcdn.com/react-faux-dom/dist/ReactFauxDOM.min.js
 
 ## Usage
 
+Basic usage with [D3][] and ES6 modules:
+
+```
+  import React from 'react';
+  import * as d3 from 'd3';
+  import Faux from 'react-faux-dom';
+
+  const MyReactClass = React.createClass({
+
+    mixins: [Faux.mixins.core, Faux.mixins.anim],
+
+    getInitialState() {
+      return {
+        chart: 'loading...'
+      }
+    },
+
+    componentDidMount() {
+      const faux = this.connectFauxDOM('div.renderedD3', 'chart')
+      const component = this;
+
+      d3.select(faux)
+        .append('div')
+        .html('Hello World!');
+
+      this.animateFauxDOM(800)
+    },
+
+    render() {
+      return (
+        <div>
+          <h2>Here is some fancy data:</h2>
+          <div className="renderedD3">
+            {this.state.chart}
+          </div>
+        </div>
+      )
+    }
+
+  });
+
+  export default MyReactClass;
+```
+
+### More usage info:
```

Comment:
IMHO for a relative newbie to both D3 and React (like myself) a self-explanatory end-to-end example that you can copy-paste and start playing with immediately is really what you need front and center in the README.

So I left mixins and animation on purpose there: assuming that people will need them sooner or later. I would may be add a comment to the `animateFauxDom` line stating that you will likely need this later and leaving a link to the example file...

But do let me know what you prefer and I'll implement your other comments and update the PR accordingly.